src/uwtools/config/tools.py

```diff
@@ -112,8 +112,10 @@ def _realize_config_check_depths(input_obj: Config, output_format: str) -> None:
     :param output_format: The output format:
     :raises: UWConfigError on excessive input-config depth.
     """
-    if (output_format == FORMAT.ini and input_obj.depth > 2) or (
-        output_format == FORMAT.nml and input_obj.depth != 2
+    if (
+        (output_format == FORMAT.ini and input_obj.depth > 2)
+        or (output_format == FORMAT.nml and input_obj.depth != 2)
+        or (output_format == FORMAT.sh and input_obj.depth != 1)
```

Comment:
Technically this is a different ticket to account for this, but it fit in here

Reply:
Shouldn't we make sure that INI format is also exactly 2?

Reply:
Yep, updating that as part of fixing ini to 2 everywhere, thanks!